lib/rules/href-no-hash.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "href-no-hash",
+  "selector": "a[href]",
+  "enabled": false,
+  "tags": [
+    "best-practice"
+  ],
+  "metadata": {
+    "description": "Ensures an anchors href value is not just #",
+    "help": "Anchors with href value that is just # should probably be a button"
```

Comment:
How about the help reading: "Anchors must only be used as links and must therefore have an href value that is a valid reference"